diff --git a/android/build.gradle b/android/build.gradle
index 41b43599a9af6ef1c170047d39db1f3baa6ca682..50126eb4e1606247d20e1aa7fb71c4008bf13b45 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -247,7 +247,7 @@ dependencies {
     extractLibs "com.microsoft.onnxruntime:onnxruntime-android:latest.integration@aar"
   }
 
-  if (VersionNumber.parse(REACT_NATIVE_VERSION) < VersionNumber.parse("0.71")) {
+  if (REACT_NATIVE_MINOR_VERSION < 71) {
     extractLibs "com.facebook.fbjni:fbjni:+:headers"
     extractLibs "com.facebook.fbjni:fbjni:+"
   }
diff --git a/unimodule.json b/unimodule.json
deleted file mode 100644
index c2bd3c0f071c0cd1ee9ccc88c7491906aed289d4..0000000000000000000000000000000000000000
